Kitodo/src/main/resources/hibernate.cfg.xml

```diff
@@ -38,11 +38,6 @@
         <property name="hibernate.c3p0.acquire_increment">1</property>
```

Comment:
Are not this two properties also related to it?

Reply:
No. This parameters are used and should not removed. Removed dependency contains only a debug class and a Oracle utils class which we did not need.